code/__DEFINES/subsystems.dm

```diff
@@ -88,6 +88,9 @@
 ///Call qdel on the atom after intialization
 #define INITIALIZE_HINT_QDEL 2
 
+///Call qdel with a force of TRUE after initialization
+#define INITIALIZE_HINT_QDEL_FORCE 3
```

Comment:
I think the `force` stuff is a gimmick when basically all of our QDEL_LETMELIVE are wrong. What are you using this for?

Reply:
I'm using it for that colossus hell item, the one that persists items

Reply:
like all the let me lives are from stupid lavaland shit bro its awful

Reply:
Correct